source/settings/plugin/imap-compress-plugin.rst

```diff
@@ -15,3 +17,25 @@ Configuration:
   protocol imap {
     mail_plugins = $mail_plugins imap_zlib
   }
+
+  plugin {
+    ## NOTE! Setting was called imap_zlib_compression_level before 2.3.15
+    ## imap_compress_<algorithm>_level = value
+    ##
+    # imap_compress_deflate_evel = 6
```

Comment:
* typo evel
 * usually at least in example-config these would be just a single `#` and the setting line as `#imap_compress..` without space after `#`
 * The default is algorithm-specific, so not 6
 * Probably default shouldn't be changed anyway .. would it be overall easier to just not have this example?